Makefile

```diff
@@ -320,83 +306,81 @@ generate-operator-api: api/v1/operator/openapi.yaml ## Generate cilium-operator
 generate-hubble-api: api/v1/flow/flow.proto api/v1/peer/peer.proto api/v1/observer/observer.proto api/v1/relay/relay.proto ## Generate hubble proto Go sources.
 	$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C api/v1
 
+define DEEP_PACKAGES
+github.com/cilium/cilium/pkg/aws/types
+github.com/cilium/cilium/pkg/azure/types
+github.com/cilium/cilium/pkg/ipam/types
+github.com/cilium/cilium/pkg/alibabacloud/types
+github.com/cilium/cilium/pkg/maps/auth
+github.com/cilium/cilium/pkg/maps/ctmap
+github.com/cilium/cilium/pkg/maps/encrypt
+github.com/cilium/cilium/pkg/maps/eppolicymap
+github.com/cilium/cilium/pkg/maps/eventsmap
+github.com/cilium/cilium/pkg/maps/fragmap
+github.com/cilium/cilium/pkg/maps/ipcache
+github.com/cilium/cilium/pkg/maps/ipmasq
+github.com/cilium/cilium/pkg/maps/lbmap
+github.com/cilium/cilium/pkg/maps/lxcmap
+github.com/cilium/cilium/pkg/maps/metricsmap
+github.com/cilium/cilium/pkg/maps/nat
+github.com/cilium/cilium/pkg/maps/neighborsmap
+github.com/cilium/cilium/pkg/maps/policymap
+github.com/cilium/cilium/pkg/maps/signalmap
+github.com/cilium/cilium/pkg/maps/sockmap
+github.com/cilium/cilium/pkg/maps/srv6map
+github.com/cilium/cilium/pkg/maps/tunnel
+github.com/cilium/cilium/pkg/maps/vtep
+github.com/cilium/cilium/pkg/node/types
+github.com/cilium/cilium/pkg/policy/api
+github.com/cilium/cilium/pkg/policy/api/kafka
+github.com/cilium/cilium/pkg/service/store
+github.com/cilium/cilium/pkg/aws/eni/types
+github.com/cilium/cilium/pkg/alibabacloud/eni/types
+github.com/cilium/cilium/api/v1/models
+github.com/cilium/cilium/pkg/bpf
+github.com/cilium/cilium/pkg/k8s
+github.com/cilium/cilium/pkg/labels
+github.com/cilium/cilium/pkg/loadbalancer
+github.com/cilium/cilium/pkg/tuple
+github.com/cilium/cilium/pkg/recorder
+github.com/cilium/cilium/pkg/k8s/types
+github.com/cilium/cilium/pkg/k8s/slim/k8s/api/discovery/v1beta1
+github.com/cilium/cilium/pkg/k8s/slim/k8s/api/discovery/v1
+github.com/cilium/cilium/pkg/k8s/slim/k8s/api/networking/v1
+github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1
+github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/apiextensions/v1
+github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/util/intstr
+github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1
+github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1beta1
+github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/labels
+github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2alpha1
+github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2
+endef
```

Comment:
I wonder if it might be nicer to define all of these in separate Makefiles (`Makefile.vars`) to more cleanly separate the data vs. the implementations? /cc @christarazi

Reply:
Oh, I guess they get deleted later anyway.